packages/slate-editor/src/components/Menu/Dropdown.tsx

```diff
@@ -1,31 +1,35 @@
 import classNames from 'classnames';
-import type { FunctionComponent, ReactNode } from 'react';
+import type { ComponentType, FunctionComponent } from 'react';
 import React from 'react';
 import type { DropdownProps } from 'react-bootstrap';
 import { Dropdown as BootstrapDropdown, MenuItem } from 'react-bootstrap';
 
-import './Dropdown.scss';
+import styles from './Dropdown.module.scss';
 
-export interface Option<Value extends string> {
-    hidden?: boolean;
-    label: string;
-    render?: (option: Option<Value>) => ReactNode;
-    value: Value;
-}
+export namespace Dropdown {
+    export interface Option<Value extends string> {
+        hidden?: boolean;
+        label: string;
+        value: Value;
+    }
 
-export interface Props<Value extends string> extends Omit<DropdownProps, 'onChange'> {
-    onChange: (value: Value) => void;
-    options: Option<Value>[];
-    value?: Value;
+    export interface Props<Value extends string> extends Omit<DropdownProps, 'onChange'> {
```

Comment:
Is it new approach that we should follow from now?

Reply:
Yes. No. I don't know. Let's talk :) 

I wanted to group the dropdown with its related types, and tried this approach. It's nicer than prefixing all the types with `Dropdown`. Nicer than ES namespaces (though not native). 

What do you think? Yay or neigh?

Reply:
I would like to give it a try, for now i do not see disadvantages 😊